recipes/openssl/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version="1.0.2h" %}
+
+package:
+  name: openssl
+  version: {{ version }}
+
+source:
+  fn: openssl-{{ version }}.tar.gz
+  url: http://www.openssl.org/source/openssl-{{ version }}.tar.gz
+  sha256: 1d4007e53aad94a5b2002fe045ee7bb0b3d98f1a47f8b2bc851dcd1c74332919
+
+build:
+  number: 0
+  no_link: lib/libcrypto.so.1.0.0        # [linux]
+  no_link: lib/libcrypto.1.0.0.dylib     # [osx]
+  detect_binary_files_with_prefix: True  # [unix]
+  has_prefix_files:                      # [unix]
+    - bin/c_rehash                       # [unix]
+    - include/openssl/opensslconf.h      # [unix]
+    - lib/pkgconfig/libcrypto.pc         # [unix]
+    - lib/pkgconfig/libssl.pc            # [unix]
+    - lib/pkgconfig/openssl.pc           # [unix]
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+
+requirements:
+  build:
+    # for feature activation
+    - python
+    - nasm               # [win32]
+    - perl     >=5.20    # [win]
+
+test:
+  requires:
+    - python   2.7.*   # [win and py27]
+    - python   3.4.*   # [win and py34]
+    - python   3.5.*   # [win and py35]
+  imports:
+    - ssl
+
+about:
+  home: http://www.openssl.org/
+  license: Apache-style (http://www.openssl.org/source/license.html)
+  license_family: Apache
+  license_file: LICENSE
+  summary: OpenSSL is an open-source implementation of the SSL and TLS protocols
+
+extra:
+  recipe-maintainers:
+    - msarahan
```

Comment:
Folks, you're all invited to this party.  Now's your last chance to run away - or just say OK.